Automattic/jetpack
# Calypsoify: Fix typo in SCSS variable

#### Changes proposed in this Pull Request:

* Update incorrect SCSS variable `$muriel-grey-0` to `$muriel-gray-0` to prevent build errors with Calypsoify styles.

#### Testing instructions:

* Switch to this PR and `yarn run build`. Check for build errors in the console (there should be none).
* Navigate to `/wp-admin/plugins.php?calypsoify=1` You should see the proper Calypsoify styles. 

#### Proposed changelog entry for your changes:
* Calypsoify: Fix typo in SCSS variable
